Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.kt

```diff
@@ -0,0 +1,1073 @@
+            }
+            builder.setMultiChoiceItems(
+                R.array.classicButtons, wiiClassicEnabledButtons
+            ) { _: DialogInterface?, indexSelected: Int, isChecked: Boolean ->
+                BooleanSetting.valueOf(classicSettingBase + indexSelected)
+                    .setBoolean(settings, isChecked)
+            }
+        } else {
+            val wiiEnabledButtons = BooleanArray(11)
+            val wiiSettingBase = "MAIN_BUTTON_TOGGLE_WII_"
+
+            for (i in wiiEnabledButtons.indices) {
+                wiiEnabledButtons[i] = BooleanSetting.valueOf(wiiSettingBase + i).boolean
+            }
+            if (currentController == InputOverlay.OVERLAY_WIIMOTE_NUNCHUK) {
+                builder.setMultiChoiceItems(
+                    R.array.nunchukButtons, wiiEnabledButtons
+                ) { _: DialogInterface?, indexSelected: Int, isChecked: Boolean ->
+                    BooleanSetting
+                        .valueOf(wiiSettingBase + indexSelected).setBoolean(settings, isChecked)
+                }
+            } else {
+                builder.setMultiChoiceItems(
+                    R.array.wiimoteButtons, wiiEnabledButtons
+                ) { _: DialogInterface?, indexSelected: Int, isChecked: Boolean ->
+                    BooleanSetting
+                        .valueOf(wiiSettingBase + indexSelected).setBoolean(settings, isChecked)
+                }
+            }
+        }
+
+        builder
+            .setNeutralButton(R.string.emulation_toggle_all) { _: DialogInterface?, _: Int ->
+                emulationFragment!!.toggleInputOverlayVisibility(settings)
+            }
+            .setPositiveButton(R.string.ok) { _: DialogInterface?, _: Int -> emulationFragment?.refreshInputOverlay() }
+            .show()
+    }
+
+    private fun chooseDoubleTapButton() {
+        val currentValue = IntSetting.MAIN_DOUBLE_TAP_BUTTON.int
+
+        val buttonList =
+            if (InputOverlay.getConfiguredControllerType() == InputOverlay.OVERLAY_WIIMOTE_CLASSIC) R.array.doubleTapWithClassic else R.array.doubleTap
+
+        var checkedItem = -1
+        val itemCount = resources.getStringArray(buttonList).size
+        for (i in 0 until itemCount) {
+            if (InputOverlayPointer.DOUBLE_TAP_OPTIONS[i] == currentValue) {
+                checkedItem = i
+                break
+            }
+        }
+
+        MaterialAlertDialogBuilder(this)
+            .setSingleChoiceItems(buttonList, checkedItem) { _: DialogInterface?, which: Int ->
+                IntSetting.MAIN_DOUBLE_TAP_BUTTON.setInt(
+                    settings,
+                    InputOverlayPointer.DOUBLE_TAP_OPTIONS[which]
+                )
+            }
+            .setPositiveButton(R.string.ok) { _: DialogInterface?, _: Int ->
+                emulationFragment?.initInputPointer()
+            }
+            .show()
+    }
+
+    @SuppressLint("SetTextI18n")
```

Comment:
This should not be suppressed. There is an actual i18n issue here, albeit a minor one.